react/features/stream-effects/blur/JitsiStreamBlurEffect.js

```diff
@@ -22,6 +22,10 @@ export default class JitsiStreamBlurEffect {
     _maskFrameTimerWorker: Worker;
     _maskInProgress: boolean;
```

Comment:
Check if we are still using all of these variables, and let's drop the ones we don't.